_dev/apps/ui/src/components/merchant-center-account/merchant-center-account-card.vue

```diff
@@ -713,6 +714,11 @@ export default {
         && this.selectedMcaDetails.name !== null
         && this.selectedMcaDetails.name !== undefined;
     },
+    hasMcaCreatedFromModule() {
+      const isFromModule = this.mcaSelectionOptions?.findIndex((option) => 'aggregatorId' in option);
```

Comment:
Checking `aggregatorId` is present is a good start, and we need to check its value too. I think we can put the value in the file `_dev/apps/ui/.env`.

If the aggregator ID is 381504106, then the merchant has already an account created via our MCA.

Reply:
Yes, you may go for it